base_import_security_group/models/models.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Anubía, soluciones en la nube,SL (http://www.anubia.es)
+# Copyright 2017 Onestein (http://www.onestein.eu)
+# Copyright 2021 ArcheTI (http://www.archeti.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+from lxml import etree
+from odoo import api, models, fields
+
+_logger = logging.getLogger(__name__)
+
+
+class IrModel(models.Model):
+    _inherit = 'ir.model'
+
+    import_group_id = fields.Many2one(
+        comodel_name='res.groups')
+
+
+class Base(models.AbstractModel):
+    _inherit = 'base'
+
+    def allow_import(self):
```

Comment:
Not blocking but in general functions that dont have to be called directly through RPC, should be prefixed with `_`